### ISSUE TITLE
А как узнать направление цепочки Cloud?

### ISSUE THREAD

**Issue by Igor731** [sbprov2/SBProV2#205]

**Comment by sbprov2** [comment]
По цвету рамки, либо включите bid-ask в настройках

**Comment by Igor731** [comment]
Cluster Border имеете ввиду? Попробовал - не добился желаемого.

**Comment by sbprov2** [comment]
нет, когда вы два раза кликаете на эллипсе, выводится информация в рамке.
А bid-ask включается в настройках Cloud

**Comment by Igor731** [comment]
Про два клика не знал. А почему она пустая. Еще не доработали?

**Comment by sbprov2** [comment]
Можно скриншот? Что именно пустое?

**Comment by sbprov2** [comment]
<img width="1079" alt="default" src="https://cloud.githubusercontent.com/assets/16720120/16589531/41dfd980-42dc-11e6-9043-d6d066098152.PNG">

**Comment by sbprov2** [comment]
Возможно у вас стоит цвет шрифта в кластерах такой же как фона графика?

**Comment by Igor731** [comment]
Да. Понял.

**Comment by sm0ll** [comment]
Истинное направление цепочки можно определить только после последующего движения. В чистом видим в моменте эта информация ничего не несет в себе.

**Comment by Igor731** [comment]
Что значит "истинное"?

**Comment by sm0ll** [comment]
В моменте цепочка может указывать направление как на покупку так и на продажу (иметь либо красную либо зеленую рамку), но это не значит что цена именно туда и пойдет. Только после того как состоится движение, можно сказать куда были направленны основные сделки. Поэтому цвет рамки никак не означает куда пойдет цена - она может пойти, например после зеленого цвета и вверх и вниз.

**Comment by Igor731** [comment]
Я спрашивал не про направление движения цены. Я спрашивал про направление сделки, реализовавшейся  в виде цепочки.
